packages/fhir-router/src/fhirrouter.ts

```diff
@@ -12,6 +12,7 @@ export type FhirRequest = {
   body: any;
   params: Record<string, string>;
   query: Record<string, string>;
+  options?: Record<string, any>;
```

Comment:
Can we move `options` to the `FhirRouter` constructor?  That way we don't have to pass it in on every request.

In the future, we could explore having per-request options, but this one makes more sense as a one-time setting.